opengrok-tools/pom.xml

```diff
@@ -280,6 +280,8 @@ Portions Copyright (c) 2017-2018, 2020, Chris Fraire <cfraire@me.com>.
                             <arguments>
                                 <argument>-m</argument>
                                 <argument>pylint</argument>
+                                <argument>--max-line-length</argument>
+                                <argument>119</argument>
```

Comment:
Why 119? I thought that the rule is 120 inclusive.

Reply:
The default was:

```
  --max-line-length n   Maximum allowed line length for the entirety of this run. (Default: 79)
```

so I believe it is 120 including the newline.